### DISCUSSION TITLE
Hide <changefreq> and <priority> from sitemap.xml

### DISCUSSION THREAD

**Discussion by nicolotor** [iamvishnusankar/next-sitemap#592] [Q&A]
Hello Guys! is it possible to remove `<changefreq>` and `<priority>` from the sitemap automatically? Thank you!

<img width="406" alt="Screenshot 2023-03-03 at 12 40 05" src="https://user-images.githubusercontent.com/66954203/222723137-c8abca4a-8dff-445f-943f-872f5d9f895b.png">

**Comment by iamvishnusankar** [comment]
@nicolotor Use a [custom transformer function](https://github.com/iamvishnusankar/next-sitemap#custom-transformation-function)

**Reply by nicolotor** [reply]
Thank you @iamvishnusankar 
since I only need the path I inserted it like this:

`transform: async (config, path) => {
        return {
            loc: path,
        };
    },
`

but if I remove the `config`  parameter it will crush, is there a way to fix this? 

<img width="601" alt="Screenshot 2023-03-03 at 15 20 11" src="https://user-images.githubusercontent.com/66954203/222758327-f69d10f7-15f7-42c4-a058-f5eb5e7f0b30.png">
